src/Engine/Party.h

```diff
@@ -76,7 +76,9 @@ enum PARTY_FLAGS_1 : int32_t {
     PARTY_FLAGS_1_ALERT_RED_OR_YELLOW = 0x0030,
     PARTY_FLAGS_1_STANDING_ON_WATER = 0x0080,
     PARTY_FLAGS_1_LANDING = 0x0100,
-    PARTY_FLAGS_1_BURNING = 0x0200
+    PARTY_FLAGS_1_BURNING = 0x0200,
+
+    PARTY_FLAGS_1_JUMPING = 0x0400
```

Comment:
Let's add a comment that this one was introduced in OE and was not present in vanilla